### PR TITLE
install py3

### PR DESCRIPTION
**Pull request by marqh**
update install guide with python 3 compatibility

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -68,8 +68,7 @@ you may need to install the development packages (look for a "-dev"
 postfix) in addition to the core packages.  
 
 python 2.7 or later (http://www.python.org/)
-    Iris requires Python 2.7 or later, but is not currently compatible with
-    Python 3.
+    Iris requires Python 2.7 or later.
```
Comment (ajdawson) [inline INSTALL]: Is this technically true? We only test against 2.7 and 3.4, and I thought there was some reason for stating we only support these versions (@rhattersley, @qulogic?). In any case I think this needs to be more specific so users immediately realise we they can use Iris on Python 3.

Comment (QuLogic) [inline INSTALL]: 3.3 is left out because it's just old. 3.5 cannot yet be supported until Cartopy supports it.

Comment (rhattersley) [inline INSTALL]: The simplest solution is just: `Iris requires Python 2.7 or Python 3.4`.

**Comment by rhattersley** [comment]
I've re-started the tests to see if the recent Cartopy 0.13.1 release fixes things.... keep those fingers crossed!

**Pull request merged**